src/Integration.Vsix/SonarLintDaemonPackage.cs

```diff
@@ -112,6 +114,15 @@ private async Task InitAsync()
             logger?.WriteLine(Strings.Daemon_InitializationComplete);
         }
 
+        /// <summary>
+        /// This migration should be performed before initializing other services, independent if a solution or a folder is opened.
```

Comment:
I feel that this comment should be above the method usage as it's a warning about the order.